.gitignore

```diff
@@ -9,3 +9,7 @@ docs/source/gallery/**
 docs/source/benchmarks/**
 .tox/
 .python-version
+.pre-commit-config.yaml
+graphix/sim/graphix.code-workspace
+
+.vscode/settings.json
```

Comment:
do we need to specify them?

Reply:
Just my local files that I wanted to ignore.